README.md

```diff
@@ -98,6 +106,19 @@ override this in your `.bashrc` or `.profile`:
 
     export HADOOP_USER_NAME=username
 
+
+Kerberos support
+----------------
+Authentication via Kerberos (and authentication only) is supported.
+
+The binary will check the default locations for your kerberos and hadoop configurations. These can be overridden via environment variables `HADOOP_KRB_CONF`, and `HADOOP_CONF_DIR`.
```

Comment:
The library does this now too, right?

Reply:
Currently building the kerberos client and setting it in the options still happens in `main.go`.

`client.go` has a few `NewX` functions: not sure if they should all check for Kerberos? Or just `New()`

OTOH, library users who need Kerberos support could build their options struct: I have no opinion on this but don't want to mess things up in `client.go` too much :)